tests/conftest.py

```diff
@@ -1,6 +1,10 @@
 import pytest
 import os
-from garak import _config
+from garak import _config, _plugins
+
+# force a local cache file to exist when this top level import is loaded
+if not os.path.isfile(_plugins.PluginCache._user_plugin_cache_filename):
+    _plugins.PluginCache.instance()
```

Comment:
Does this not trigger [_plugins.py#L63](https://github.com/jmartin-tech/garak/blob/9438858605634bcb915e66368e5e70f817566762/garak/_plugins.py#L63) ?


```
    def _load_plugin_cache(self):
        assert os.path.exists(
            self._plugin_cache_filename
        ), f"{self._plugin_cache_filename} is missing or corrupt."
```